src/shared/lib/path.ts

```diff
@@ -54,9 +54,7 @@ export function getPathTarget(path: string, os_path_style: OsPathStyle = OS_DEFA
   */
   if (os_path_style == OsPathStyle.posix) {
     if (path == OsPathSeparator.posix) return path
-    if (path.endsWith(OsPathSeparator.posix)) {
-      path = path.slice(0, path.length - 1)
-    }
+    path = path.replace(/\/+$/g, "");
```

Comment:
Should became better if there was usage of `osPathSeparator.posix` but AFAIK there is no such nice solutions